pages/quests/quest2.mdx

```diff
@@ -223,6 +223,14 @@ Run the below command to fetch latest news:
 aos> Send({ Target = ao.id, Action = "Get-News" })
 ```
 
+To run the Cron-Handler:
+```
+aos <process_name> --cron 2-seconds
```

Comment:
Our heading is getting the latest data in 4 hours. Then why have you entered 2 seconds here?